<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smartHome</class>
 <widget class="QMainWindow" name="smartHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>559</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>778</width>
    <height>559</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>778</width>
    <height>559</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>778</width>
     <height>559</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>778</width>
     <height>559</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>45</height>
     </size>
    </property>
    <widget class="QWidget" name="master">
     <attribute name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/image/wali.jpg</normaloff>:/image/wali.jpg</iconset>
     </attribute>
     <attribute name="title">
      <string>主界面</string>
     </attribute>
     <widget class="QLabel" name="wali_image">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>111</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/welcome.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="ParityBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>380</y>
        <width>72</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Space</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Even</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mark</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="text_recData">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>64</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>接收数据</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_send">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>231</width>
        <height>401</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_dataBit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>340</y>
        <width>45</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>数据位</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_receive">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>60</y>
        <width>231</width>
        <height>401</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="BaudRateBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>320</y>
        <width>72</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>72</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="StopBitBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>360</y>
        <width>72</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="text_baud">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>45</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_stopBit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>45</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>停止位</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_port">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>300</y>
        <width>45</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>串口号</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_parity">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>380</y>
        <width>45</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>校检位</string>
      </property>
     </widget>
     <widget class="QComboBox" name="FlowControlBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>400</y>
        <width>72</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Soft</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hard</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="Btn_clearSend">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>30</y>
        <width>75</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>29</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>清空发送</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_flowControl">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>400</y>
        <width>45</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>流控制</string>
      </property>
     </widget>
     <widget class="QComboBox" name="DataBitBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>340</y>
        <width>72</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="SerialPortBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>300</y>
        <width>72</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>72</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_startTrans">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>430</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开始传输</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_sendData">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>64</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>发送数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_clearRec">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>75</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>29</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>清空接收</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;欢迎来到瓦力家居控制系统&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;点击上方图标进入该区域查看并进行控制&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;在下方选择栏中设置连接端口的信息&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;并可在左边文本框中查看传输的数据内容&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="movie_wave">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="livingroom">
     <attribute name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/image/livingroom.png</normaloff>:/image/livingroom.png</iconset>
     </attribute>
     <attribute name="title">
      <string>客厅</string>
     </attribute>
     <widget class="QPushButton" name="Btn_clear">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>440</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>启动清洁</string>
      </property>
     </widget>
     <widget class="QLabel" name="image_clear">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>380</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/clear1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="image_light">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/liv_light1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="image_audio2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>50</y>
        <width>51</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/voice1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="image_TV">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/tv1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="image_audio1">
      <property name="geometry">
       <rect>
        <x>243</x>
        <y>50</y>
        <width>51</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/voice1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QwtThermo" name="humidity">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>51</width>
        <height>111</height>
       </rect>
      </property>
      <property name="scaleMaxMinor">
       <number>10</number>
      </property>
      <property name="alarmEnabled">
       <bool>true</bool>
      </property>
      <property name="value">
       <double>55.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="text_humi">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>120</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>当前湿度</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_TV">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>130</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开电视</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_sound">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开音响</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_temp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>当前温度</string>
      </property>
     </widget>
     <widget class="QwtThermo" name="temperature">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>111</height>
       </rect>
      </property>
      <property name="upperBound">
       <double>50.000000000000000</double>
      </property>
      <property name="scaleMaxMajor">
       <number>5</number>
      </property>
      <property name="scaleMaxMinor">
       <number>10</number>
      </property>
      <property name="alarmEnabled">
       <bool>true</bool>
      </property>
      <property name="value">
       <double>26.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_light">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>320</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开电灯</string>
      </property>
     </widget>
     <widget class="QLabel" name="image_air">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>180</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/air1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_air">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>250</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开空调</string>
      </property>
     </widget>
     <widget class="QLabel" name="image_window">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>320</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/window1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSlider" name="slider_window">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>400</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bedroom">
     <attribute name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/image/bedroom.png</normaloff>:/image/bedroom.png</iconset>
     </attribute>
     <attribute name="title">
      <string>卧室</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="bookroom">
     <attribute name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/image/bookroom.png</normaloff>:/image/bookroom.png</iconset>
     </attribute>
     <attribute name="title">
      <string>书房</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="kitchen">
     <attribute name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/image/kitchen.png</normaloff>:/image/kitchen.png</iconset>
     </attribute>
     <attribute name="title">
      <string>厨房</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="bathroom">
     <attribute name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/image/bathroom.png</normaloff>:/image/bathroom.png</iconset>
     </attribute>
     <attribute name="title">
      <string>卫生间</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="outdoor">
     <attribute name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/image/outdoor.png</normaloff>:/image/outdoor.png</iconset>
     </attribute>
     <attribute name="title">
      <string>门外</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
